src/ifp/src/InitFloorplan.tcl

```diff
@@ -267,6 +267,11 @@ proc insert_tiecells { args } {
 
 namespace eval ifp {
 
+proc to_dbu { coord } {
```

Comment:
why is this needed? isn't coord already in microns? it seems like a lot of conversions

Reply:
No it is in STA distance units.  Likely that is microns but it doesn't have to be.  It is inconsistent with other commands that use microns but consistent with other commands the user sta units for resistance/capacitance.

Reply:
The point of this is to allow the python interface to use dbu

Reply:
that seems odd, is there a chance a user will not know this and get undesireable results?

Reply:
I haven't added any conversion, just moved them around between c++ & tcl

Reply:
sure, I just noticed the double conversion and thought that was strange since most of the other tools juts use ord::microns_to_dbu and doesn't have this interim step. I was just wondering. If it's correct then it's correct.

Reply:
I think it points to an inconsistency but I'll tackle that separately.  I don't see the needs for distance units to come from Liberty and would prefer to use microns everywhere.  For res/cap/time using sta units makes more sense.